backend/src/main/java/de/capstonemarc/backend/baby/BabyService.java

```diff
@@ -12,7 +14,16 @@ public class BabyService {
 
     public Baby addBaby(NewBaby newBaby) {
         String id = babyUtils.generateUUID();
-        Baby baby = new Baby(id, newBaby.name());
+        Baby baby = new Baby(id,
+                newBaby.name(),
+                newBaby.birthday(),
+                newBaby.weight(),
+                newBaby.height(),
+                newBaby.gender());
         return babyRepository.save(baby);
```

Comment:
I guess this block is necessary to generate the ID for the new baby?
Consider using  @With in your record and then just set Baby baby = newBaby.withId(id)